proto/private/prototest/testing.go

```diff
@@ -100,5 +100,5 @@ func AssertContainsElement[V any](t TestingT, list []V, element V, opts ...cmp.O
 		}
 	}
 
-	t.Fatalf("assertion failed: list does not contain element\n--- list\n%#v\n--- element: %#v", list, element)
+	t.Fatalf("assertion failed: list does not contain element\n--- list\n%+v\n--- element: %+v", list, element)
```

Comment:
This change was needed to avoid triggering the race detector.

The internal/catalog/internal/controllers packages define well known status conditions that the controllers will add to resources. The protobuf system will eventually fill in the unexported size cache bits of the protobuf struct. For some reason the `%#v` formatting does some copying around of data in a way that copies an atomic value without actually using atomics (`reflect.typedmemmove`).

Somehow switching to the `%+v` format specifier avoids that copy and prevents data races.